2020-01-Winter/5_graphs/README.md

```diff
@@ -138,21 +164,45 @@ Go to [Top](#top)
 <!-- Don't remove -->
 <a name="s3"/>
 
-### 3. SOLUTION 3 TODO :bug:
-
-Source: TODO :bug:
-
-#### Naive/Simple Solution 
-
-TODO :bug:
-
-#### Optimal Solution
-
-TODO :bug:
+### 3. Is Graph Bipartite
+
+Source: [Leetcode](https://leetcode.com/problems/is-graph-bipartite/)
+
+#### Solution
+
+````
```

Comment:
Adding the language name here will make sure your code has syntax highlighting, which makes it easier to read.
```suggestion
````java
```